packages/account/src/Components/form-body-section/form-body-section.tsx

```diff
@@ -25,11 +30,7 @@ const FormBodySection = ({ children, has_side_note, side_note }) => {
             </div>
         );
     }
-    return children;
-};
-FormBodySection.prototype = {
-    children: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
-    has_side_note: PropTypes.bool,
-    side_note: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+    return children as JSX.Element;
```

Comment:
**Nit:** Rather than typecasting, may be it is better to return `<React.Fragment>{children}</React.Fragment>`

Reply:
@likhith-deriv it brakes tests. I also changed the same part of code for other components in other PRs